src/theory/sets/theory_sets_private.cpp

```diff
@@ -2155,8 +2150,24 @@ void TheorySetsPrivate::preRegisterTerm(TNode node)
   }
 }
 
+// Here, we need to be notified when a universe set occurs in our input,
+// before preprocessing and simplification takes place. If the sets-ext option
+// is not set, we throw an exception.
+// This function is a no-op otherwise.
+Node TheorySetsPrivate::expandDefinition(LogicRequest &logicRequest, Node n) {
```

Comment:
1. Move this comment to the header file.
2. Explain why you need to be notified early if all you're doing is checking for an option being set.  If you check later, what goes wrong?
3. Mention that you are using the expandDefinition interface in a more general way and reference the open issue.

Reply:
Should we maintain a one-to-one mapping with git hub issues and comments that mention them?  I referenced the issue in theory.h.

Reply:
Generally, I thinik it's fine to have TODO's with references to issues in the code anywhere something might be affected by that issue.  These don't all have to be mentioned in the issue itself - we can have scripts that make sure those TODO's are cleaned up when the issue is closed - I'll bring this up in the next meeting.  So a reference in theory.h is good (although I don't see it - is it part of another pending commit?)  But a reference here is fine as well so that we can fix the comment if we change the name of expandDefinitions.

Reply:
It was part of the same commit: https://github.com/CVC4/CVC4/pull/1070/commits/d7cad64d2ed0133352e98acde174b39850585d28#diff-5d5a157c15ed54a6debc0add9dbe8abfR428

I've expanded the comment and moved it to the .h file in my latest commit.